taxonomy/management/commands/populate_job_names.py

```diff
@@ -0,0 +1,71 @@
+"""
+Management command for populating missing job names by extracting from EMSI.
+"""
+
+import logging
+
+from django.core.management.base import BaseCommand, CommandError
+
+from taxonomy.constants import get_lookup_query_filter
+from taxonomy.emsi_client import EMSIJobsApiClient
+from taxonomy.enums import RankingFacet
+from taxonomy.exceptions import TaxonomyAPIError
+from taxonomy.models import Job
+from taxonomy.utils import chunked_queryset
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    command for populating missing job names
+
+    While creating Job object we only populate its id. We need to get all the Job in the system that do no have names.
+
+    Example usage:
+        $ # Update the existing job names.
+        $ ./manage.py populate_job_names
+        """
+    help = 'populating missing Job names in the system.'
+
+    def _update_job(self, job_bucket):
```

Comment:
I think we already have a helping method with a similar name which may add confusion, how about we make this one more specific and change it to `_update_job_name` instead?